tools/run_tests/run_tests_matrix.py

```diff
@@ -130,6 +130,15 @@ def _create_test_jobs(extra_args=[], inner_jobs=_DEFAULT_INNER_JOBS):
                              extra_args=extra_args,
                              inner_jobs=inner_jobs)
 
+  for compiler in ['python3.4', 'python3.5', 'python3.6', 'pypy']:
```

Comment:
@jtattermusch In principle, I agree with you. In practice, we've seen some discrepancies between different 3.x variants in their behavior. With that said, I agree with the premise of running the full-fledged suite on the latest 2.7 and latest stable 3.x (3.6 atm) and running portability on other versions. Does portability test only make the build? or does it also run a small subset of the test suite? If it runs a small subset of the suite, is that subset easily configurable?